src/AST/FunctionExtensions.cs

```diff
@@ -54,7 +57,8 @@ public static class FunctionExtensions
                     @params.Add(new Parameter
                         {
                             QualifiedType = pointer,
-                            Name = "instance"
+                            Name = "instance",
+                            Namespace = param.Namespace
```

Comment:
Set to the function this param is added to. In this one and all other cases where there's a function.